app/src/main/res/values/strings.xml

```diff
@@ -559,6 +559,9 @@ be temporary, and you can try again later.</li>
     <string name="settings_cookies_dialog_title">Clear all cookies and site data</string>
     <string name="settings_cookies_dialog_content">Selecting ‘Clear Now’ will clear all cookies and site data stored by Firefox for Fire TV. This may sign you out of websites and remove offline web content.</string>
 
+    <!-- Home Tile context menu strings -->
+    <string name="home_tile_context_menu_remove">Remove</string>
```

Comment:
Is this rebased off of master? I think this string is already in this string file, so you don't need to add a new one - just use the one in the file. IIRC you reviewed the code to add this string for me.